CMakeLists.txt

```diff
@@ -4,9 +4,7 @@ project( EOSIO )
 include(CTest) # suppresses DartConfiguration.tcl error
 enable_testing()
 
-if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
-   set(CMAKE_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}")
-elseif ("${CMAKE_INSTALL_PREFIX}" STREQUAL "/usr/local")
+if ("${CMAKE_INSTALL_PREFIX}" STREQUAL "/usr/local")
    message(WARNING "CMAKE_INSTALL_PREFIX is explicitly set to /usr/local. This is not recommended.")
```

Comment:
The word `explicitly` should be removed from the message, since `CMAKE_INSTALL_PREFIX` could be set to it by default.